app/src/main/res/values/themes.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <!-- Base application theme. -->
+    <style name="Theme.MP" parent="Theme.Backbone">
```

Comment:
I was surprised we didn't have an app theme yet.  Are UI colors/styles hard-coded in all the views?